### PR TITLE
added final form & text field  validation

### PR DESCRIPTION
**Pull request by arizona-kh**

### CODE REVIEW

File: src/components/SignUp.js [hunk 1]
```diff
@@ -117,23 +160,46 @@ export default function SignUp() {
...
+            disabled={submitting || pristine}
           >
             Sign Up
           </Button>
+          </Grid>
+          </Grid>
           <Grid container justify="flex-end">
             <Grid item>
-              <Link href="#" variant="body2">
+              <Link href="/signin" variant="body2">
```
Comment (mentikora) [inline src/components/SignUp.js:194]: There is a good practice to keep routes in a sinle file and import when you need them.

For example: 
Create routes.js file with one export constant "routes"
`export const routes = {
  signin: '/signin',
}`

Then import this file and use like {routes.signin}

File: src/components/SignUp.js [hunk 2]
```diff
@@ -117,23 +160,46 @@ export default function SignUp() {
               />
             </Grid>
           </Grid>
+          <Grid 
+            container 
+            spacing={2}
+            alignItems="center">
+          <Grid item xs={6}>
+          <Button
```
Comment (mentikora) [inline src/components/SignUp.js:168]: add tab

File: src/components/SignUp.js [hunk 3]
```diff
@@ -117,23 +160,46 @@ export default function SignUp() {
...
+            variant="contained"
+            color="default"
+            className={classes.submit}
+            href="/"
+          >
+            Back Home
+          </Button>
+          </Grid>
+          <Grid item xs={6}>
           <Button
```
Comment (mentikora) [inline src/components/SignUp.js:180]: add tab

File: src/components/SignUp.js (hunk 1 above)
Comment (arizona-kh) [reply src/components/SignUp.js:194]: круто, цього я не знала

**Pull request merged**